# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/binutils/sparc64-pagesize.patch
# Copyright (C) 2023 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- binutils-2.44/bfd/elf64-sparc.c.vanilla	2025-02-02 21:20:30.324000293 +0100
+++ binutils-2.44/bfd/elf64-sparc.c	2025-02-02 21:20:34.460000295 +0100
@@ -889,7 +889,7 @@
 #define TARGET_BIG_NAME	"elf64-sparc"
 #define ELF_ARCH	bfd_arch_sparc
 #define ELF_TARGET_ID	SPARC_ELF_DATA
-#define ELF_MAXPAGESIZE 0x100000
+#define ELF_MAXPAGESIZE 0x2000
 #define ELF_COMMONPAGESIZE 0x2000
 
 /* This is the official ABI value.  */
